### PR TITLE
Restructure 3

### PR DESCRIPTION
**Pull request by rmariani**

### CODE REVIEW

File: concepts/azimuth.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
++++
+title = "Azimuth"
+weight = 4
+template = "doc.html"."
```
Comment (matildepark) [inline concepts/azimuth.md]: This lil dot typo will cause a fail on build.

File: tutorials/_index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## [Hoon Tutorial](@/docs/learn/hoon-tutorial/_index.md)
+
+Hoon is a high-level, statically typed, functional programming language, custom-designed for use with Urbit. For a functional language it has a surprisingly imperative style, and mastering it doesn't require knowing any advanced mathematics. The Hoon Tutorial is the best place to start if you want to learn Urbit.
+
+## [Nock](@/docs/learn/nock/_index.md)
+
+Nock is the low-level functional programming language that is interpreted by Vere. You don't need to know Nock to write Hoon, or even to work on most parts of Vere.  But if you want to understand the foundations of Urbit and are mathematically-inclined, dive right in.
+
+## [Vere](@/docs/learn/vere/_index.md)
```
Comment (matildepark) [inline tutorials/_index.md]: Please amend all the relative links on these pages to their new locations — otherwise, as is will cause a fail on build.

Comment (rmariani) [reply tutorials/_index.md]: aye aye

**Comment by matildepark** [comment]
Once this is merged I'll have to do a pass and amend some more relative links — @/operations.md, @/install.md and the like no longer exist and will also cause a fail on build.

**Comment by rmariani** [comment]
Ok, pushed a new commit that fixes all those `@/docs` links

File: reference/hoon-errors.md [hunk 3]
```diff
@@ -20,28 +20,28 @@ a correct Hoon file.  These values are always correct.
 Usually, the line and column tell you everything you need to
 know.  But the worst-case scenario for a syntax error is that,
 somewhere above, you've confused Hoon's tall form by using the
-wrong fanout for a rune.  For example, `%+` ([**cenlus**](@/docs/hoon/hoon-expressions/rune/cen.md#cenlus),
-a function call whose sample is a cell) has three subhoons:
+wrong fanout for a rune.  For example, `%+` ([**cenlus**](@/docs/reference/hoon-expressions/rune/cen.md#cenlus),
+a function call whose sample is a cell) has three subreferences:
 
-```hoon
+```reference
```
Comment (matildepark) [inline reference/hoon-errors.md]: This is just one example, but a lot of mentions of the word 'hoon' are now 'reference'. Not just the syntax highlight metadata. Try the find/replace again with `/docs/hoon` -> `/docs/reference/`, etc.

**Comment by rmariani** [comment]
Ok ok I guessed that I might have had grep-replace wrong. I made a backup in anticipation

**Comment by matildepark** [comment]
Sorry to commit to master behind your back — some docs-based relative links to the new IA on urbit.org side are preventing its build while design work implementation is happening, so had to be done ASAP ... you'll have to pull it in.

**Comment by rmariani** [comment]
Handled instead by https://github.com/urbit/docs/pull/654

**Pull request closed without merging**